index.html

```diff
@@ -108,18 +108,17 @@ <h2>Tonic</h2>
                     <ul class="ul-list">
                         <button class="buttons"> <li>HTML</li></button>
                         <button class="buttons"><li>CSS</li></button>
```

Comment:
Nice work on your project cards✨. But to make them look more like the figma design, let's adjust a few things:

Your design:
<img width="240" alt="image" src="https://github.com/Salarhabibi-amir/Salarhabibi-amir.github.io/assets/37116322/fa7bda01-1bd0-471d-99bc-9c0a00764a4b">

Figma design:
<img width="165" alt="image" src="https://github.com/Salarhabibi-amir/Salarhabibi-amir.github.io/assets/37116322/e4d6be0e-cea5-4e21-aed8-d46f2eb01549">

A few things to note:
- The background color used in the figma design for the modal is closer to `#d7dae1`. Please update your design to use the same
- Kindly make sure all the content of the modal is left aligned, especially the paragraph text
- Kindly update the background color of the technology list items to be `#ebebff`
- Please update your icons for the see live and see source buttons to be the same as the figma design. You can try exporting the images from figma and styling them to fit your project